functions/_pure_prompt_ssh_user.fish

```diff
@@ -0,0 +1,9 @@
+function _pure_prompt_ssh_user
+    set --local username (whoami) # current user name
```

Comment:
In this instance, my preference is to use `-l` instead of `--local`. My rationale is that one declares variables often enough to warrant the exception. Feel free to disagree! 😄

Reply:
Again, this change has been made here by @rafaelrinaldi: https://github.com/rafaelrinaldi/pure/commit/2998cf802f50d7ada2bc968a1ec367616b044c70. I just trying to stick with one approach throughout all the code. In my opinion, short version is preferable, though.

Reply:
Prefer long format options as they are more explicit than the cryptic 1-letter form.